src/pages/LendingStats/LendingStatsPage.vue

```diff
@@ -1,8 +1,10 @@
 <template>
 	<www-page>
-		<the-my-kiva-secondary-menu slot="secondary" />
+		<template #secondary>
+			<the-my-kiva-secondary-menu />
+		</template>
 		<div class="row page-content">
-			<the-portfolio-tertiary-menu slot="tertiary" class="show-for-large" />
+			<the-portfolio-tertiary-menu class="show-for-large" />
```

Comment:
Should this be
```
<template #tertiary>
	<the-portfolio-tertiary-menu class="show-for-large" />
</template>
```
?

Reply:
Technically yes. But that breaks the page. I don't believe that this page was actually using the tertiary slot on www-page. Probably because it was being rendered inside of the default slot. I don't think any component actually is, and it can actually be removed. 

The problem is that the slot on www-page is right above the default slot, so adding the menu as the first thing on the default slot displays it correctly. 

The correct syntax for lending stats should be:
```
		<template #secondary>
			...
		</template>
		<template #tertiary>
			...
		</template>
                <div class="row page-content">
                ...
                </div>
```

But what happens in that case is the tertiary slot, which has grid classes, gets displayed outside of page-content and breaks the page. 

The original markup:
```
	<the-my-kiva-secondary-menu slot="secondary" />
		<div class="row page-content">
			<the-portfolio-tertiary-menu slot="tertiary" class="show-for-large" />
                ...
                </div>

```
Was just rendering the secondary slot and the default slot. You can verify this  by seeing how the "row page-content" div is interweaved with the slots. 

You can see similar behavior in other pages that have a tertiary menu, they are just added to the default slot so they would render correctly (legal sub pages)